### PR TITLE
147 update playerlist if player exits session

### LINKED ISSUE #147: Update playerlist if player exits session

### PR DESCRIPTION
**Pull request by ivarhe**
closes #147

### CODE REVIEW

**Comment by ivarhe** [comment]
Added a temporary solution for a dynamic score-system based on who answers first, since we don't have any logic for that in the backend yet.

Open for discussion, is this a bad idea? 🤔

File: web/app/components/AnswerView.tsx [hunk 1]
```diff
@@ -5,8 +5,7 @@ import { useGameContext } from '~/context/game/GameContext'
 export default function AnswerView(): ReactElement {
     const { state } = useGameContext()
 
-    const onNameClick = () => {
-    }
+    const onNameClick = () => { /* void */}
```
Comment (perkynades) [inline web/app/components/AnswerView.tsx:8]: Is this function needed?, if not delete

**Review by perkynades — APPROVED** [review]
if (pr) approve

**Comment by perkynades** [comment]
> Added a temporary solution for a dynamic score-system based on who answers first, since we don't have any logic for that in the backend yet.
> 
> Open for discussion, is this a bad idea? 🤔

If the backend decides if a question is correctly answered or not, then i think the backend should handle giving a score.

**Review by MatiasGranHenriksen — APPROVED** [review]

File: web/app/components/AnswerView.tsx (hunk 1 above)
Comment (ivarhe) [reply web/app/components/AnswerView.tsx:8]: Doesn't look like it, i just made the linter happy

**Comment by ivarhe** [comment]
> > Added a temporary solution for a dynamic score-system based on who answers first, since we don't have any logic for that in the backend yet.
> > Open for discussion, is this a bad idea? 🤔
> 
> If the backend decides if a question is correctly answered or not, then i think the backend should handle giving a score.

Agree, but for now i don't think the backend keeps track of the answers the players give. So they can't give score based on position

**Pull request merged**